packages/emulation/libretro-beetle-pcfx/package.mk

```diff
@@ -51,11 +51,12 @@ make_target() {
     imx6)
       make platform=armv7-cortexa9-neon-hardfloat
       ;;
-    WeTek_Play|WeTek_Core)
-      make platform=armv7-cortexa9-neon-hardfloat
-      ;;
-    Odroid_C2|WeTek_Hub|WeTek_Play_2)
-      make platform=aarch64
+    WeTek_Play|WeTek_Core|Odroid_C2|WeTek_Hub|WeTek_Play_2)
+      if [ $TARGET_ARCH = 'aarch64' ];then
```

Comment:
`[ "$TARGET_ARCH" = "aarch64" ]` for consistency?

Reply:
Ahhh I see. Thanks I'm gonna fix it.